src/index.spec.js

```diff
@@ -633,49 +633,43 @@ describe('Dropzone', () => {
       expect(child).toHaveProp('isDragReject', false)
     })
 
-    it('should add valid files to rejected files on a multple drop when multiple false', done => {
+    it('should add valid files to rejected files on a multple drop when multiple false', async () => {
       const dropzone = mount(<Dropzone accept="image/*" onDrop={dropSpy} multiple={false} />)
       dropzone.simulate('drop', { dataTransfer: { files: images } })
-      setTimeout(() => {
-        const rejected = dropSpy.firstCall.args[0]
-        expect(rejected.length).toEqual(1)
-        done()
-      }, 0)
+      await null
```

Comment:
@okonet this was the way to make it 'await' thus allowing the stub to resolve before doing the assertions